mesonbuild/compilers/rust.py

```diff
@@ -220,11 +220,16 @@ def use_linker_args(cls, linker: str, version: str) -> T.List[str]:
     # use_linker_args method instead.
 
     def get_options(self) -> MutableKeyedOptionDictType:
-        return dict((self.create_option(options.UserComboOption,
-                                        self.form_compileropt_key('std'),
-                                        'Rust edition to use',
-                                        ['none', '2015', '2018', '2021', '2024'],
-                                        'none'),))
+        opts = super().get_options()
+
+        key = self.form_compileropt_key('std')
+        opts[key] = options.UserComboOption(
+            self.make_option_name(key),
+            'Rust edition to use',
+            ['none', '2015', '2018', '2021'],
```

Comment:
2024 is dropped?

Reply:
Rebase error. 2024 wasn't in Meson when this was originally written :D